src/health/health.controller.spec.ts

```diff
@@ -39,17 +39,4 @@ describe('HealthController', () => {
       });
     });
   });
-
```

Comment:
This test is failing because there is no setup for the test, it was rely on other files.